.github/workflows/terraform-tests.yml

```diff
@@ -40,6 +40,17 @@ jobs:
     - name: Checkout Localstack
       uses: actions/checkout@v3
 
+    # FIXME: pinned because of https://github.com/hashicorp/terraform-provider-aws/issues/27049
+    - name: Install terraform
+      uses: little-core-labs/install-terraform@v2.0.0
```

Comment:
What's the reason of using this over the official `hashicorp/setup-terraform@v2` action? :thinking:

Reply:
This was the first thing that came up when i googled for it 🤷 happy to use the official one